app/services/save_auction.rb

```diff
@@ -4,10 +4,20 @@ def initialize(auction)
   end
 
   def perform
-    auction.save
+    saved = auction.save
+    schedule_auction_ended_job(saved)
```

Comment:
thoughts on passing `auction` and checking if it is persisted down on L18? I know `save` returns true or false but truthiness and falsiness sometimes get me into weird places. Clearer to check for actual values / methods like `persisted?`